torchvision/models/mobilenet.py

```diff
@@ -117,7 +111,21 @@ def mobilenet_v2(pretrained=False, progress=True, **kwargs):
         pretrained (bool): If True, returns a model pre-trained on ImageNet
         progress (bool): If True, displays a progress bar of the download to stderr
     """
-    model = MobileNetV2(**kwargs)
+    default_setting = [
```

Comment:
Would it be simpler to do
```python
if inverted_residual_setting is None:
    inverted_residual_setting = ...
```
?

Reply:
Thank you for the suggestion i have changed it that way